draft-ietf-quic-transport.md

```diff
@@ -1131,6 +1131,13 @@ client MUST terminate with a QUIC_VERSION_NEGOTIATION_MISMATCH error code if
 version negotiation occurred but it would have selected a different version
 based on the value of the supported_versions list.
 
+The position and size of the version validation fields in the transport
+parameters MUST NOT change between different versions of QUIC.  If a version of
+QUIC needs to define a new format for transport parameters, it MUST define and
+use a new TLS extension rather than redefine the layout of the existing
```

Comment:
I feel like it would be clearer if you expand on the attack you’re trying to mitigate.  Something like:
“When a peer accepts multiple versions of the handshake, it can potentially interpret the handshake as defined by any of the versions it supports.  Since the QUIC protocol relies on security of the handshake for authenticating the version, the position and the format of the version field in the handshake MUST be consistent across all QUIC versions supported by the client.”